### DISCUSSION TITLE
Love Wiki|Docs

### DISCUSSION THREAD

**Discussion by stormpaul** [Zavy86/WikiDocs#79] [Show and tell]
Congratulations on creating a super simple flat file wiki.  Something that finally gets the end user to focus on what is important, writing the documentation.  I have tried so many CMS, both with and without database (I wanted flat file) and they all have some kind of issue - difficult to install, unstable, time consuming build process, too many features that take away from the focus of writing documentation, etc - the list goes on.  

This is so super simple to get up and running and once done, it's just there, doing what it is supposed to do.  It's so cut down and focused you cannot even create user accounts, just one password to view and one password to edit, great work!

There are only 2 (fairly minor) features that I feel are missing:

- A right side bar with content of headings on the page that is being viewed (created from H2 and H3 headings on the page) to make navigation within the page easier
- A back to top button to prevent sometimes long scrolling up

I am still using it even without these, however they would make the experience a little better if they were available in any future release.

Thank you.

**Comment by Zavy86** [comment]
Hi Paul, thank you so much! <3
I saw that you also opened an issue regarding the two points mentioned..
These are two reasonable requests, I think they can be developed, but in this period I'm a bit messed up, I'll see when I'll be able to take care of them.
Put a star in the repository and sooner or later your wishes will be fulfilled!
:-)
